eos_data_distribution/ndn/command.py

```diff
@@ -47,6 +47,17 @@ def makeCommandInterest (*args, **kwargs):
     'flagbit': 'setFlagBit'
 }
 
+def addNextHop(faceUri, name, cost=0, *args, **kwargs):
+    face = Face(faceUri)
```

Comment:
we're not, we're connecting via a GLibUnixFace to NFD telling it to connect to another machine,
has been implemented differently and removed.